source/documentation/common/install/assembly-Installing_Hosts_for_RHV.adoc

```diff
@@ -64,8 +64,8 @@ include::proc-Installing_Cockpit_on_RHEL_Hosts.adoc[leveloffset=+2]
 
 include::con-Recommended_practices_for_configuring_host_networks.adoc[leveloffset=+1]
 
-ifdef::SHE_cockpit_deploy,SHE_cli_deploy[]
+ifdef::SHE_cli_deploy[]
```

Comment:
@emarcusRH I don't understand why you're removing the conditionalization here. The document still exists.

Reply:
We are removing all mention of Cockpit SHE installation, including the document. 
Dev have wanted to get rid of this since the beginning of the RHV 4.4  cycle
I thought it might be prudent to get rid of the conditional refs too